src/components/About.jsx

```diff
@@ -19,7 +20,18 @@ function About() {
           muted
           style={{ position: "absolute", objectFit: "cover", zIndex: "-1" }}
         />
-
+ <Card.Body>
+            <Card.Title>Inaia Martin</Card.Title>
+            <Card.Text>
+              I am a dedicated professional looking for a front end developer
+              role following my completion of the University of Denver’s
+              FullStack BootCamp. I’m a dynamic learner (I speak native
+              Portuguese, English and Spanish), and hard working (I completed
+              DU’s bootcamp while working full time), and I’m excited to learn
+              and grow in this new career. Currently working as a Freelance to
+              improve my skills.
+            </Card.Text>
+          </Card.Body>
         <Card className="Cardaboutme">

```

Comment:
best practice convention in CSS is [`kebab-case`](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/#use-hyphen-delimited-strings) classes so this should be
```jsx
className="card-about-me"
```